docs/components/button.md

```diff
@@ -0,0 +1,29 @@
+| Status      | ![Carbon Button](https://img.shields.io/npm/dw/carbon-components.svg)<br>![Carbon Button](https://img.shields.io/npm/v/carbon-components.svg) | ![Carbon Button React](https://img.shields.io/npm/dw/carbon-components-react.svg)<br>![Carbon Button React](https://img.shields.io/npm/v/carbon-components-react.svg) | ![Carbon Button Angular](https://img.shields.io/npm/dw/carbon-components-angular.svg)<br>![Carbon Button Angular](https://img.shields.io/npm/v/carbon-components-angular.svg)<br>![Status new](https://img.shields.io/badge/status-new-blueviolet.svg) | ![Carbon Button Vue](https://img.shields.io/npm/dw/@carbon/vue.svg)<br>![Carbon Button Vue](https://img.shields.io/npm/v/@carbon/vue.svg) |
+| Maintainers | [@carbon-design-system/core-team]()                                                                                                           | [@carbon-design-system/core-team]()                                                                                                                                   | [@carbon-design-system/angular-team](), [@user1]()                                                                                                                                                                                                     | [@user2]()                                                                                                                                |
+
+## Specification test results
+
+|           | Type     | Vanilla                                                              | React                                                                | Angular                                                                       | Vue                                                                  |
+| --------- | -------- | -------------------------------------------------------------------- | -------------------------------------------------------------------- | ----------------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| General   | -        | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg)          | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) |
+| Primary   | variant  | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg)          | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) |
```

Comment:
## spec numbering
confused on the numbers on these. What does a `1` ( ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) ) mean here vs a `9` ( ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg)) above?

## data and inheritance 

Thinking about where all this data will come from. For our components, we track component-level and demo-level data and store that data in a [component].config.js file (just like what's in each component in carbon-components). This structure allows the testing-data to be inherited by demos from their parent component.

Example (from our `core`) image component of how we track browser support at the component level:

```
const support = {
  browsers: {
    chrome: 'true',
    firefox: 'true',
    safari: 'true',
    edge: 'true',
    ie11: 'true',
  },
...
```

but then those can be overridden at the `demo` level:

```
const picture = {
  name: 'picture',
  label: 'Picture',
  support: {
    browsers: {
      ie11: {
        content:
          'The `picture` scenario does not meet all PAL browser requirements.' +
          ' This method can only be used for applications that do not need to support IE11.',
        url:
          'https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#Browser_compatibility',
        support: 'mixed',
      },
    },
  },
};
```

## demo wcag rating

How does the `spec` rating work vs the `wcag` rating? ie...if `Accordion` is rated at wcag-2.0-aa, then by passing all spec tests would a component and/or demo be considered `aa`? What if a given demo or component had a better rating of `aaa`?

I'm wondering if the wcag rating could be tracked as a separate data point at the data & component & framework level:

```
Button:
  Spec: WCAG-aa
  Vanilla: spec-passing (WCAG-aa)
  Vue: primary button @ WCAG-aaa, 
      component @ WCAG-aa means:
      vue-button/primary=wcag-aaa, but vue-button=wcag-aa, but _both_ passing spec
```

|           | Type     | Vanilla                                                              | React                                                                | Angular                                                                       | Vue                                                                  |
| --------- | -------- | -------------------------------------------------------------------- | -------------------------------------------------------------------- | ----------------------------------------------------------------------------- | -------------------------------------------------------------------- |
| General   | -        | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg)          | ![](https://img.shields.io/badge/spec-%E2%9C%94%209-brightgreen.svg) |
| Primary   | variant  | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg)          | ![](https://img.shields.io/badge/spec-%E2%9C%94%201-brightgreen.svg) ![WCAG 2.0 Level AAA](https://img.shields.io/badge/spec-aaa-blue.svg)|

Reply:
I'm working on component configs (living in package.json files once each component is individually published) and revisiting component directory structure now that we have a better understanding of where we want to go compared to a couple months ago. Let's go over that next sync-up! Still a WIP over here.